.github/CODEOWNERS

```diff
@@ -1,12 +1,6 @@
-# iOS
-/platform/ios/ @mapbox/maps-ios-reviewers
-/platform/ios/docs/ @mapbox/docs
-
-# macOS
-/platform/macos/ @1ec5 @mapbox/maps-ios-reviewers
-/platform/macos/docs @mapbox/docs
-
-
-# Darwin
-/platform/darwin/ @1ec5 @mapbox/maps-ios-reviewers
+* @mapbox/gl-native
 
+# Apple
+/platform/ios/ @mapbox/maps-ios-reviewers
+/platform/macos/ @mapbox/maps-ios-reviewers
+/platform/darwin/ @mapbox/maps-ios-reviewers
```

Comment:
I don’t expect this repository to have any new macOS-only files for the foreseeable future, but if that ever happens, @mapbox/maps-ios-reviewers can feel free to reach out to me for a closer look.